ipyleaflet/leaflet.py

```diff
@@ -2,16 +2,19 @@
 # Distributed under the terms of the Modified BSD License.
 #
 
+from ast import In
 import copy
 import asyncio
 import json
 import xyzservices
 from datetime import date, timedelta
 from math import isnan
+import branca.colormap as cm
+from branca.colormap import linear
```

Comment:
This makes ipyleaflet dependent on branca now. So we either need to add it as a dependency in the `setup.py`, or we can find a way to import it lazily for the `ColormapControl` usage.

Reply:
I will add a dependency in the setup.py